### PR TITLE
Added reference to GC Notfiy `post` with ckan site url without schema…

### PR DESCRIPTION
**Pull request by JVickery-TBS**
… and port.

### CODE REVIEW

File: ckanext/gcnotify/mailer.py [hunk 1]
```diff
@@ -162,10 +162,13 @@ def get_request_body(recipient,
                     personalisation):
     # type: (str, str, dict) -> dict|None
 
+    ref, sep, tail = config.get('ckan.site_url', '').replace('https://', '').replace('http://', '').partition(':') # type: str
```
Comment (RabiaSajjad) [inline ckanext/gcnotify/mailer.py]: @JVickery-TBS I am expecting simply the url e.g. https://github.com/open-data/og/blob/58866274a12d86920c31169e6e0be7dc0919e36c/modules/custom/gcnotify/src/Plugin/WebformHandler/GCNotifyEmailHandler.php#L117. And within the controller, I am resolving domain name https://github.com/open-data/og/blob/58866274a12d86920c31169e6e0be7dc0919e36c/modules/custom/gcnotify/src/Controller/GCNotifyRestController.php#L124. Let me know if it helps.

Comment (JVickery-TBS) [reply ckanext/gcnotify/mailer.py]: @RabiaSajjad Okay it should be simplified now. I tested it on staging, so you should be able to see the `STAGING REGISTRY` in the table in production.

**Comment by JVickery-TBS** [comment]
@RabiaSajjad Okay, it is now set up to use the request uri as the reference.

**Comment by RabiaSajjad** [comment]
Parfait, merci beaucoup @JVickery-TBS

**Pull request merged**